### ISSUE TITLE
Question about definition of label images in train_multi script

### ISSUE THREAD

**Issue by b1a0** [uncbiag/uniGradICON#28]
I have a few rather specific questions about a some code lines in the train_multi.py script. 

First, the function `make_network` does not seem to have a parameter `use_label=True`. Is this intentional? Should it be called without this specification?
Secondly, in line 178 you define 
 
`moving_image, fixed_image, moving_image, fixed_image = data_augmenter(moving_image, fixed_image, moving_label, fixed_label)`

From what I understand this means that while at first you have separate images for moving_image and moving_label, you then overwrite them with being the same? I am unsure if this is intentional, as this fixed_image then also takes the position of fixed_label in `train_kernel` (as defined in line 136) and called from line 179:

`train_kernel(optimizer, net, moving_image, fixed_image, moving_image, fixed_image, writer, iteration)`

Could you clarify if these things are intentional and/or if I need to correct the above things to get the training script running correctly? Thank you in advance!

**Comment by basardemir** [comment]
Hello @b1a0,

Thank you for pointing out the issues in the training code. These errors were unintentional and occurred during the code-cleaning process. I have created a pull request to fix that errors.

- The make_network function has a use_label parameter, but the pip package has not been updated yet. We will update it as soon as possible.

https://github.com/uncbiag/uniGradICON/blob/a9b1310a80ed9d4fb72fdcf3cfa5cea9d2dec2ce/src/unigradicon/__init__.py#L171

<br>

- I have fixed the label parameter issue in both the data_augmenter and train_kernel function calls.
